static/gsApp/views/subscriptionPage/reservedUsageChart.tsx

```diff
@@ -463,9 +463,6 @@ function ReservedUsageChart({
   const categoryStats = usageStats[category];
   const isReservedBudgetCategory =
     subscription.reservedBudgetCategories?.includes(category) ?? false;
-  if (isReservedBudgetCategory) {
-    displayMode = 'cost';
-  }
```

Comment:
should we keep the override for sales-led customers since they don't have the toggle option?